tests_e2e/orchestrator/lib/agent_test_suite_combinator.py

```diff
@@ -260,8 +266,8 @@ def create_environment_list(self) -> List[Dict[str, Any]]:
 
         return environments
 
-    def create_existing_vm_environment(self) -> Dict[str, Any]:
-        loader = AgentTestLoader(self.runbook.test_suites, self.runbook.cloud)
+    def create_existing_vm_environment(self, test_suites: List[str]) -> Dict[str, Any]:
+        loader = AgentTestLoader(test_suites, self.runbook.cloud)
```

Comment:
how is loader used in create_existing_vm_environment and create_existing_vmss_environment? I don't see it being used anywhere here

Reply:
we get the test suites from it, it's the last property in the return value

`c_test_suites": loader.test_suites,`